templates/publisher/account-snaps.html

```diff
@@ -71,11 +71,26 @@ <h3 class="p-heading--five">Join the community</h3>
             <h3 class="p-heading--four">Congratulations!</h3>
           </div>
           <div class="col-6">
-            <p>
-              You've just released a snap to the "{{ snaps[snap].latest_revisions[0].channels[0] }}" channel!<br />
-              Want to improve the listing in stores? You can add an icon and screenshots.<br />
-              <a href="/account/snaps/{{ snap }}/market" class="p-button--neutral">Go to market listing</a>
-            </p>
+            {% if snaps[snap].latest_revisions[0].channels[0] %}
+              <p>
+                You've just released {{ snap }} to the "{{ snaps[snap].latest_revisions[0].channels[0] }}" channel!
+              </p>
+              <ul>
+            {% else %}
+              <p>
+                You've just uploaded {{ snap }}!
+              </p>
+              <ul>
+                <li>
+                  Happy your snap is ready for users?
+                  <a href="https://docs.snapcraft.io/build-snaps/release" class="p-link--external" target="_blank">Release it</a>
+                </li>
+            {% endif %}
+              <li>
+                Want to improve the listing in stores?
```

Comment:
Should it be 

> store/s?

Reply:
Each snap can technically be in multiple stores, so I think 'stores' is better in this context